src/encoded/static/components/regulome_search.js

```diff
@@ -1305,6 +1361,26 @@ class RegulomeSearch extends React.Component {
                                         </div>
                                     </React.Fragment>
                                 : null}
+                                {Object.keys(hitSnps).map(rsid =>
+                                    <div className="notification-line" key={rsid}>
+                                        <div className="notification-label">{rsid}</div>
+                                        <div className="notification">
+                                            <div>
+                                                {sortedPopulations[rsid].slice(0, 3).map(
+                                                    population => <div>{`${hitSnps[rsid][population]} (${population})`}</div>
+                                                )}
+                                            </div>
+                                            {sortedPopulations[rsid].length > 3 && this.state.showMoreFreqs ?
```

Comment:
Why 3?

Reply:
In general, arbitrary. Specifically, I got this "feeling" from dbSNP like this: https://www.ncbi.nlm.nih.gov/snp/rs3768324. Any good suggestions?

Reply:
3 is fine, but naked integer parameter is a little wonky.  Not sure what the protocol is for JS constants.